lib/localeapp/cli/install.rb

```diff
@@ -79,13 +81,25 @@ def write_config_file
           write_rails_config
         end
 
+        def write_apikey_to_dotenv
+          @output.puts "NOTICE: API key saved to .env"
```

Comment:
Not yet: I would write the file first ;-)
It may fail, but you'd have this message on standard output.